### PR TITLE
Optimize flips between 1 and 0

### PR DESCRIPTION
**Pull request by AZero13**
We can solve this with a bit flip rather than + 1 mod 2.

### CODE REVIEW

File: src/coreclr/vm/eetwain.cpp [hunk 1]
```diff
@@ -2644,10 +2644,8 @@ unsigned scanArgRegTableI(PTR_CBYTE    table,
                 {
                     thisPtrReg = REGI_NA;
                 }
-                if  (iptrRegs & regMask)
-                {
-                    iptrRegs &= ~regMask;
-                }
```
Comment (huoyaoyuan) [inline src/coreclr/vm/eetwain.cpp:2650]: This change looks valid, but not directly related to other changes and the topic of this PR.

**Comment by JulieLeeMSFT** [comment]
Ping @EgorBo, please review this PR.

**Review by EgorBo — APPROVED** [review]
Thanks!

**Comment by EgorBo** [comment]
Please fix formatting jobs

**Pull request closed without merging**